src/features/classification/__tests__/Classification.test.js

```diff
@@ -77,16 +79,32 @@ describe('features/classification/Classification', () => {
         expect(wrapper.exists('[data-testid="classification-modifiedby"]')).toBe(false);
     });
 
-    test('should render classification last modified information when provided and message style is inline', () => {
-        const wrapper = getWrapper({
-            name: 'Confidential',
-            definition: 'fubar',
-            messageStyle: 'inline',
-            modifiedAt: '2020-07-16T00:51:10.000Z',
-            modifiedBy: 'A User',
-        });
-        expect(wrapper.exists('[data-testid="classification-modifiedby"]')).toBe(true);
-    });
+    test.each`
+        useAppliedByLabel | expectedActionLabelId
+        ${undefined}      | ${messages.appliedByText.id}
+        ${true}           | ${messages.appliedByText.id}
+        ${false}          | ${messages.importedFromText.id}
+    `(
+        'should render classification last modified information with expected action label when provided and message style is inline',
```

Comment:
nit: Could we incorporate `useAppliedByLabel` and `expectedActionLabelId` to the message so that it's easier to figure out what is being tested from the command line?